lib/api/objectPut.js

```diff
@@ -82,15 +82,18 @@ function objectPut(authInfo, request, streamingV4Params, log, callback) {
                 }
                 return next();
             },
-            function createCipherBundle(next) {
-                const serverSideEncryption = bucket.getServerSideEncryption();
-                if (!checkObjectEncryption.isValidSSES3(request,
-                    serverSideEncryption)) {
-                    return next(invalidSSEError);
+            next => getObjectSSEConfiguration(headers, bucket, log, (err, sseConfig) => {
```

Comment:
To be consistent with the other steps:

```suggestion
            function getSSEConfig(next) {
                getObjectSSEConfiguration(headers, bucket, log, (err, sseConfig) => {
```